Core/src/IfcGeometryConverter/ProfileConverter.h

```diff
@@ -942,27 +866,27 @@ namespace OpenInfraPlatform {
 					const double b = t_shape->FlangeWidth * length_factor;
 					const double tw = t_shape->WebThickness * length_factor * 0.5;
 					const double tf = t_shape->FlangeThickness * length_factor;
-
-					double r1 = 0;
+					//Redo
+					double r1 = 0.0;
 					if(t_shape->FilletRadius) {
 						r1 = t_shape->FilletRadius * length_factor;
 					}

```

Comment:
This can still be refactored to `double r1 = t_shape->FilletRadius.value_or(0.0) * length_factor;`

Reply:
Done